docs/pages/eas-update/debug.mdx

```diff
@@ -1,69 +1,87 @@
 ---
 title: Debug EAS Update
-sidebar_title: Debug
+sidebar_title: Basic
 description: Learn how to debug EAS Update.
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { Terminal } from '~/ui/components/Snippet';
 
-It's important to tell the current state of our app at any given time. We built EAS Update with this in mind. Once we know which updates are running on which builds, we can make changes so that our apps are in the state we expect and desire. This guide sets out to show how we can verify our EAS Update and expo-updates configuration so that we can find the source of problems like an app not showing a published update.
+This guide shows how to verify our configuration so that we can find the source of problems like an app not showing a published update. It's important to tell the current state of our app at any given time, so we built EAS Update with this in mind. Once we know which updates are running on which builds, we can make changes so that our apps are in the state we expect.
 
-## expo-updates configuration
+## Go to the Deployments page
 
-The expo-updates library runs inside an end-user's app and makes requests to an update server to get the latest update.
+The EAS website has a [Deployments page](https://expo.dev/accounts/[account]/projects/[project]/deployments) that shows the current state of our app. The term _deployment_ refers to a group of builds and their updates. If we've made builds and updates with EAS, you can see your project's status on the website in the Deployments tab.
```

Comment:
```suggestion
The EAS website has a [Deployments page](https://expo.dev/accounts/[account]/projects/[project]/deployments) that shows the current state of our app. The term _deployment_ refers to a group of builds and their corresponding updates. If we've made builds and updates with EAS, we can see our project's status on the website in the Deployments tab.
```

(fwiw, I don't really like the `we` voice for docs in general, but that's outside the scope of this PR. this suggestion just adheres to a consistent voice)